CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   and `SubspaceContinuous` classes
 - New mechanisms for surrogate input/output scaling configurable per class
 - `SurrogateProtocol` as an interface for user-defined surrogate architectures
+- `get_surrogate` and `posterior` methods to `Campaign`
```

Comment:
surprised you added those to campaign
While I think they can be *also* in Campaign, canonically arent they supposed to be in the recommenders?

Reply:
looks good, agree also about the posterior choice